go/db/generate_base.go

```diff
@@ -138,7 +138,7 @@ var generateSQLBase = []string{
 		  last_checked timestamp NULL DEFAULT NULL,
 		  last_seen timestamp NULL DEFAULT NULL,
 		  mysql_port smallint(5) unsigned DEFAULT NULL,
-		  count_mysql_snapshots smallint(5) unsigned NOT NULL,
+		  count_mysql_snapshots smallint(5) unsigned NOT NULL DEFAULT 0,
```

Comment:
Must change: existing SQL must never ever change. The most we can do is add a modification in `generate_patches.go`. However, modifying a column is not supported by `sqlite` so this change must unfortunately go away.

The value is implicitly `0` anyway, what is the concern?

Reply:
oh. ok. sorry i will rollback this change

Reply:
yes concern is 
count_mysql_snapshots not used when try insert into ```host_agent``` table
and sql query is failed when agent submitting